docs/output.md

```diff
@@ -12,6 +12,7 @@ of [`pixelator`](https://github.com/PixelgenTechnologies/pixelator).
 
 The pipeline consists of the following steps:
 
+- [Pipeline information](#pipeline-information) - Report metrics generated during the workflow execution
```

Comment:
best to list these steps in the order of execution

Reply:
Fixed: [0b0b2ca](https://github.com/nf-core/pixelator/pull/121/commits/0b0b2caf5aa490841bd304cf1a43a0a85efbcdd5)